screwdriver.yaml

```diff
@@ -0,0 +1,37 @@
+cache:
+    event:
+        - $SD_SOURCE_DIR/node_modules
+    pipeline:
+        - ~/.gradle
+    job:
+        test-job: [/tmp/testblah]
+jobs:
+    test-job:
+        image: node:6
+        steps:
+            - env: env
```

Comment:
can remove all these steps and just have the `ls /tmp` to show that the file is already there.

Reply:
fixed